.travis.yml

```diff
@@ -0,0 +1,20 @@
+# R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
+
+dist: trusty   # Use Ubuntu 14.04 image (instead of 12.04)
+language: R
+sudo: false
+r: 3.5
```

Comment:
why doesn't it specify 3.5.2?

Reply:
There were a few notes in the  about only using the minor version instead of the bug fix version. (Also, I tried it with 3.5.2 and it bombed.)